androidApp/src/main/java/com/example/tasky/android/agenda/components/DetailsReminderSection.kt

```diff
@@ -24,10 +24,19 @@ import com.example.tasky.android.theme.Gray
 import com.example.tasky.android.theme.Light2
 import com.example.tasky.model.agenda.AgendaItem
 
+enum class ReminderType(val milliSecond: Long) {
+    TEN_MINUTE((10 * 60 * 1000).toLong()),
+    THIRTY_MINUTE((30 * 60 * 1000).toLong()),
+    ONE_HOUR((60 * 60 * 1000).toLong()),
+    SIX_HOUR((6 * 60 * 60 * 1000).toLong()),
+    ONE_DAY((24 * 60 * 60 * 1000).toLong()),
+}
```

Comment:
Here, I would suggest making use of Kotlin’s duration class

```
import kotlin.time.Duration

enum class ReminderType((val duration: Duration) {     
  TEN_MINUTE(10.minutes),     
  THIRTY_MINUTE(30.minutes),     
  ONE_HOUR(1.hours),     
  SIX_HOUR(6.hours),     
  ONE_DAY(1.days), 
}
```

You can also convert these to a Long if you want to: `10.minutes.toLong(DurationUnit.MILLISECONDS)`